### DISCUSSION TITLE
ZBDongle-E non riesco a capire come supporta Bluetooth

### DISCUSSION THREAD

**Discussion by fsecond3bis** [indomus/forum#2212] [GENERALE - Discussioni generiche]
Ciao, sono un novizio e dopo aver letto i vostri ottimi tutorial sto configurando homeassistant con questo dongle. Tutto bene, configurato su pc Intel con debian e docker. Dalla vostra recensione pare che il chip EFR32MG21 supporti Bluetooth, ma non ho capito come come far funzionare il Bluetooth su ZBDongle-E; in altre parole homeassistant (e l'hardware del pc) vedono solo il device zigbee /dev/ttyACM0 ma nessuna scheda Bluetooth. 
Ho anche aggiornato il firmare alla versione ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20220812194326
bisogna fare qualche altro magheggio? o faccio prima a comprare un dongle usb bluetooth? :D

**Comment by snakuzzo** [comment]
Il fatto che abbia un chip per il BT non vuol dire che sia utilizzato.
Dipende anche dal firmware utilizzato probabilmente.
Dovresti cercare altri firmware per quel tipo di chip che ti permettano di utilizzarlo.

**Reply by fsecond3bis** [reply]
Grazie della risposta! da quel che ho capito googlando si può flashare un firmware bluetooth su EFR32MG21 ma si perde la funzionalità zigbee, quindi lascio perdere e compro solo sensori zigbee ;)

**Comment by ivanpass** [comment]
Salve,
anch'io mi trovo nella stessa situazione e avevo preso il nuovo dongle proprio per poterlo sfruttare con più device.
Ho trovato l'articolo seguente che sembra poter sfruttare i chip integrati su un unico bus, ma è un po' troppo tecnico per le mie competenze.
https://github.com/SiliconLabs/cpc-daemon/blob/main/readme.md
Magari se un vero smanettone ci mettesse le mani potrebbe darci qualche consiglio .

Grazie!
